Chefster.csproj

```diff
@@ -36,6 +36,7 @@
       </PackageReference>
       <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="17.10.0" />
       <PackageReference Include="MongoDB.Driver" Version="2.25.0" />
+      <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.2" />
```

Comment:
Should this be `Microsoft.EntityFrameworkCore.MySql`?

Reply:
Pomelo.EntityFrameworkCore.MySql seems to be one of the standards:
https://learn.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli

Reply:
Ope nvm then! 👍🏻